base_contact/__openerp__.py

```diff
@@ -26,19 +26,8 @@
     'website': 'http://www.openerp.com',
     'category': 'Customer Relationship Management',
     'complexity': "expert",
-    'description': """
-This module allows you to manage your contacts
```

Comment:
You might want to put this phrase in `'summary'` of in `__openerp__.py` just bellow `'name'` key